src/pydvl/value/loo/naive.py

```diff
@@ -20,8 +18,9 @@ def naive_loo(
     """
 
     values = {i: 0.0 for i in u.data.indices}
+    all_indices = set(u.data.indices)
     for i in maybe_progress(data.indices, progress):  # type: ignore
-        subset = np.setxor1d(u.data.indices, [i], assume_unique=True)
+        subset = all_indices.difference({i})
```

Comment:
Is this faster than setxor1d? I assumed that that was the reason why that was used.

Reply:
At any rate it is clearer, and speed is not of the essence here.